selectrum.el

```diff
@@ -917,13 +917,20 @@ ignores the currently selected candidate, if one exists."
       selectrum--start-of-input-marker
       selectrum--end-of-input-marker))))
 
-(defun selectrum-insert-current-candidate ()
-  "Insert current candidate into user input area."
-  (interactive)
-  (when selectrum--current-candidate-index
+(defun selectrum-insert-current-candidate (&optional arg)
+  "Insert current candidate into user input area.
+
+With optional prefix argument ARG, insert the candidate at that
+index (counting from one, clamped to fall within the candidate
+list). Zero means to select the current user input."
```

Comment:
This should be documented as well in the README, like the prefix argument for `RET`.